test/functional/feature_blockfilter_version.py

```
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# Copyright (c) 2025 The Dash Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test blockfilter index version checking."""
+
+import os
+import shutil
```

Ruff (0.12.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)